pkg/proc/types.go

```diff
@@ -149,6 +150,10 @@ func runtimeTypeToDIE(_type *Variable, dataAddr uint64) (typ godwarf.Type, kind
 
 	// go1.7 to go1.10 implementation: convert runtime._type structs to type names
 
+	if goversion.ProducerAfterOrEqual(_type.bi.Producer(), 1, 17) {
```

Comment:
Could you add a comment here explaining why we no longer need this in >= 1.17 before I merge this, please.